ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -0,0 +1,79 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+<Messages::TabPageHeader
+  @authenticated={{@message.authenticated}}
+  @pageTitle="{{if @message.isNew 'Create' 'Edit'}} message"
+  @breadcrumbs={{this.breadcrumbs}}
+/>
+
+<form id="message-create-edit-form" {{on "submit" (perform this.save)}} data-test-form="create-and-edit">
+  <div class="box is-sideless is-fullwidth is-marginless has-top-padding-s">
+    <Hds::Text::Body @tag="p" class="has-bottom-margin-l" data-test-form-subtext>
+      {{if @message.isNew "Create" "Edit"}}
+      a custom message for all users when they access a Vault system via the UI.
+    </Hds::Text::Body>
+
+    <MessageError @errorMessage={{this.errorBanner}} class="has-top-margin-s" />
+
+    {{#each @message.formFields as |attr|}}
+      {{#if (eq attr.name "linkTitle")}}
+        <div class="field has-bottom-margin-m">
+          <label for="link" class="is-label">Link
+            <span class="has-text-grey-400 has-font-weight-normal">(optional)</span></label>
+          <div class="control is-flex-between has-gap-m">
+            <Input
+              @type="text"
+              @value={{@message.linkTitle}}
+              placeholder="Display text (e.g. Learn more)"
+              id="link-title"
+              class="input"
+              data-test-field="linkTitle"
```

Comment:
I recommend making this `data-test-input="{{@attr}}"` so for testing you can use the same fillIn test selector as form field uses, same comment below!